src/SprykerEco/Zed/Payone/Business/Distributor/OrderPriceDistributor.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Distributor;
+
+use Generated\Shared\Transfer\OrderTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use SprykerEco\Zed\Payone\PayoneConfig;
+
+class OrderPriceDistributor implements OrderPriceDistributorInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     * @return \Generated\Shared\Transfer\OrderTransfer
+     */
+    public function distributeOrderPrice(OrderTransfer $orderTransfer): OrderTransfer
+    {
+        if ($orderTransfer->getPayments()->count() <= 1) {
+            return $orderTransfer;
+        }
+
+        $payonePayment = $this->findPayonePayment($orderTransfer);
+        if (!$payonePayment) {
+            return $orderTransfer;
+        }
+
+        $totals = $orderTransfer->getTotals();
+
+        $roundingError = 0;
```

Comment:
Maybe, it will be better to move this value to class const with a meaningful name.

Reply:
It's just a default `int` value which is used only once.